### PR TITLE
Addressing rocket-pool/smartnode#601 on v2 branch

### PR DESCRIPTION
**Pull request by SolezOfScience**
Porting the changes made as part of https://github.com/rocket-pool/smartnode-install/pull/135 to the v2 branch.

### CODE REVIEW

**Comment by jshufro** [comment]
I think you accidentally included an extra commit in this PR. It should just be the mev-boost issue, but has a commit intended to implement partial wallet recovery.

**Comment by SolezOfScience** [comment]
You're right - not sure how that happened. I'll fix this.

File: install/deploy/templates/mev-boost.tmpl [hunk 1]
```diff
@@ -19,6 +19,7 @@ services:
       - NETWORK={{.Network}}
       - MEV_BOOST_PORT={{.MevBoost.Port}}
       - MEV_BOOST_RELAYS={{.MevBoost.GetRelayString}}
+      - MEV_BOOST_ADDITIONAL_FLAGS={{.MevBoost.AdditionalFlags}}
```
Comment (jshufro) [inline install/deploy/templates/mev-boost.tmpl:22]: I don't see this being added to the config struct in this pr. I'm assuming it was already cherry-picked in to v2?

**Review by jshufro — APPROVED** [review]

File: install/deploy/templates/mev-boost.tmpl (hunk 1 above)
Comment (SolezOfScience) [reply install/deploy/templates/mev-boost.tmpl:22]: I think it's already being passed, unless I'm misunderstanding. [Code Link](https://github.com/rocket-pool/smartnode/blob/v2/shared/config/mev-boost-config.go#L73)

Also looks like it gets passed [here](https://github.com/rocket-pool/smartnode/blob/v2/shared/config/mev-boost-config.go#L282). I also checked the UI on my node and confirmed the parameter box is present but it won't be used without this change.

**Pull request merged**